pages/docs/manual/latest/exception.mdx

```diff
@@ -109,25 +134,234 @@ if (exit === 1) {
 
 </CodeTab>
 
-You can also make your own exceptions like you'd make a variant (exceptions need to be capitalized too).
+### `Invalid_argument`
+
+Used to check if argument is valid. This exception takes a string
```

Comment:
don't forget the `.`

Reply:
https://github.com/rescript-association/rescript-lang.org/pull/880/commits/2680345f43f67314928f06eea32ae5261ab0cf72